### PR TITLE
UVIS update

### PR DESCRIPTION
**Pull request by rosteen**
@duytnguyendtn I wanted this here so I could start working on the unit tests here instead of in sandbox, so I went ahead and did it. Also note that in the interest of having any config files in the same place, I changed the config directory layout to `config/telescope` instead of `telescope/config` (currently it's just HST)

### CODE REVIEW

File: astrogrism/grism_observation.py [hunk 1]
```diff
@@ -10,11 +7,9 @@
 import numpy as np
 import pathlib
 
-from .HST.transform_models import WFC3IRForwardGrismDispersion, \
-                                  WFC3IRBackwardGrismDispersion
-from .HST.dispersion_models import DISPXY_Extension
+from HST.transform_models import WFC3IRForwardGrismDispersion, WFC3IRBackwardGrismDispersion
+from HST.dispersion_models import DISPXY_Extension
```
Comment (duytnguyendtn) [inline astrogrism/grism_observation.py:11]:
```suggestion
from .HST.transform_models import WFC3IRForwardGrismDispersion, \
                                  WFC3IRBackwardGrismDispersion
from .HST.dispersion_models import DISPXY_Extension
```

**Review by duytnguyendtn — CHANGES_REQUESTED** [review]
Found the testing failure. The leading `.` indicated a relative (package) import. Reverting these changes allowed my pytest to pass

File: astrogrism/grism_observation.py (hunk 1 above)
Comment (rosteen) [reply astrogrism/grism_observation.py]: Ahh good call. Thanks!

**Comment by rosteen** [comment]
Test coverage will be addressed in a follow-up PR

**Comment by duytnguyendtn** [comment]
Wonderful! Should I review as is @rosteen, or are you planning on adding UVIS tests to this PR (in which case, I'll hold off)?

**Comment by rosteen** [comment]
Any objection to merging this now that CI is passing?

**Comment by duytnguyendtn** [comment]
In the midst of a review now. I doubt there will be any issues, so if it's blocking your work, go for it!

**Review by duytnguyendtn — APPROVED** [review]
The UVIS_Testing notebook works as expected with this PR. Merging!

**Pull request merged**